src/Parallel-Tests.ps1

```diff
@@ -23,7 +23,14 @@ foreach ($a in $assemblies)
 {
     $xmlName = 'xUnit-Results-' + [System.IO.Path]::GetFileNameWithoutExtension($a) + '.xml'
     $outXml = $(Join-Path $outDir $xmlName)
-    $cmdLine = $xunitRunner.FullName + ' ' + $a + ' ' + $testFilter + ' -xml ' + $outXml + ' -parallel none -noshadow -verbose' 
+	if([System.IO.Path]::GetFileNameWithoutExtension($a) -eq 'DefaultCluster.Tests')
```

Comment:
yeah make sense, I already turned the parallelizeTestCollections on in .xunit.runner.json. maybe that's why it doesn't have any time again with -parallel collection param. 

will revert this file